diograph.spec.ts

```diff
@@ -0,0 +1,275 @@
+import { IDiograph, IDiographObject, IDiory, IDioryObject } from './types'
```

Comment:
I think it would be good to split / make clear sections between different methods: addDiograph, queryDiograph, getDiory etc. But I don't know exactly how... I had that /api folder and each method there in its own file...

If it's one big file I'm afraid that not every method is getting tested properly because it seems just "there's lots of tests" although in fact the test coverage is poor...

Reply:
Yes, my original idea was that each method would be a pure function but the most methods were so simple that it felt an overkill.